src/assets/js/play_url.js

```diff
@@ -0,0 +1,50 @@
+(function (window, document) {
+  "use strict";
+
+  var EXERCISES_PATH = "/exercises";
+  var PLAY_HOST = "https://play.rust-lang.org";
+  var RUST_VERSION = "nightly";
+  var HOME_URL = window.location.protocol + "//" + window.location.host + "/";
+
+  document.addEventListener("DOMContentLoaded", function() {
```

Comment:
This will fail in IE 8. Alternative solution would be putting whole script inline, inside main HTML file, just before closing `</body>` tag. Not the nicest solution, but should work everywhere.